Fakebook.Posts/Fakebook.Posts.RestApi/Startup.cs

```diff
@@ -27,21 +29,20 @@ public class Startup {
 
         // This method gets called by the runtime. Use this method to add services to the container.
         public void ConfigureServices(IServiceCollection services) {
-
-            var connectionString = Configuration.GetConnectionString("default");
-            if (connectionString is null) {
-                throw new InvalidOperationException("No connection string 'defaualt' found.");
-            }
-
-            services.AddDbContext<FakebookPostsContext>(options =>
-            options.UseNpgsql(connectionString));
+            // setup Postgres database
+            if (Configuration["ConnectionString:default"] is string connectionString) 
+                services.AddDbContext<FakebookPostsContext>(options => options.UseNpgsql(connectionString));
+            else throw new NullReferenceException("No connection string 'default' found.");
```

Comment:
throwing nullreferenceexception is bad practice just fyi

Reply:
thanks, just read [Using Standard Exception Types](https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/using-standard-exception-types) I'll change it to `ArgumentNullException`